CHANGELOG.md

```diff
@@ -1,22 +1,44 @@
 # Cromwell Change Log
 
-## 0.20
+## 0.22
 
-* The default per-upload bytes size for GCS is now the minumum 256K
-instead of 64M. There is also an undocumented config key
-`google.upload-buffer-bytes` that allows adjusting this internal value.
+* Improved retries for Call Caching and general bug fixes.
+* Now there are configurable caching strategies for the SharedFileSystem backends (i.e. Local). This new "caching"
+  stanza will be nested inside of: backend.{your_SFS_backend}.config.filesystems.local stanza.
+  See below for detailed descriptions of each configurable key.
 
-* Updated Docker Hub hash retriever to parse json with [custom media
-types](https://github.com/docker/distribution/blob/05b0ab0/docs/spec/manifest-v2-1.md).
+```
+caching {
+    duplication-strategy: [
+    "hard-link", "soft-link", "copy"
+    ]
+
+     # Possible values: file, path
+     # "file" will compute an md5 hash of the file content.
+     # "path" will compute an md5 hash of the file path. This strategy will only be effective if the duplication-strategy (above) is set to "soft-link",
+     # in order to allow for the original file path to be hashed.
+     hashing-strategy: "file"
+
+     # When true, will check if a sibling file with the same name and the .md5 extension exists, and if it does, use the content of this file as a hash.
+     # If false or the md5 does not exist, will proceed with the above-defined hashing strategy.
+     check-sibling-md5: false
+}
+```
+* Mulitple Input JSON files can now be submitted in server mode through the existing submission endpoint: /api/workflows/:version.
+    This endpoint accepts a POST request with a multipart/form-data encoded body. You can now include multiple keys for workflow inputs.
 
-* Added a `/batch` submit endpoint that accepts a single wdl with
-multiple input files.
+        The keys below can contain optional JSON file(s) of the workflow inputs. A skeleton file can be generated from wdltool using the "inputs" subcommand.
+        NOTE: Each prcoceeding workflowInput file will override any JSON key conflicts.
 
-* The `/query` endpoint now supports querying by `id`, and submitting
-parameters as a HTTP POST.
+        workflowInputs
+        workflowInputs_2
+        workflowInputs_3
+        workflowInputs_4
+        workflowInputs_5
 
-## 0.21
+* Batched status polling of Google Jes for running jobs.
```

Comment:
But do add:

You can now limit the number of concurrent jobs for a backend by specifying the following option in the backend's config stanza:

```
backend {
  ...
  providers {
    BackendName {
      actor-factory = ...
      config {
        concurrent-job-limit = 5
```